### PR TITLE
#134 add scale-control-example

### PR DESCRIPTION
**Pull request by OpenSteveMap**
Here is a scale control example

### CODE REVIEW

File: examples/scale-control-directive/index.ts [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    <span class="input-group-addon fixed-space">maxWidth</span>
+      <input type="number" class="form-control"  [(ngModel)]="maxWidth">
+    </div><!-- /input-group -->
+    
+    <h4>Control Options</h4> 
+    <div class="input-group">
+    <span class="input-group-addon fixed-space">Position</span>
+    
+    <select  class="form-control" name="state" [(ngModel)]="position">
+        <option *ngFor="let state of states" [ngValue]="state">
```
Comment (atd-schubert) [inline examples/scale-control-directive/index.ts]: `positionStates` instead of `states`

File: examples/scale-control-directive/index.ts [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+@Component({
+    selector: 'app',
+    template
+})
+export class AppComponent implements AfterViewInit {
+    public metric: boolean = true;
+    public imperial: boolean = true;
+    // todo: public updateWhenIdle: boolean = true;
+    public maxWidth: number = 150;
+    public states: ControlPosition[]= [
```
Comment (atd-schubert) [inline examples/scale-control-directive/index.ts]: see above...

File: examples/scale-control-directive/index.ts [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+        'bottomleft' , 'bottomright', 'topleft' , 'topright'
+    ];
+    public position: ControlPosition;
+
+
+    @ViewChild(MapComponent) private mapComponent: MapComponent;
+
+    constructor() {
+        (<any>window).app = this;
+        this.position = this.states[0];
```
Comment (atd-schubert) [inline examples/scale-control-directive/index.ts]: see above...

**Review by atd-schubert — CHANGES_REQUESTED** [review]
Same as the other PR, let's use positionStates

**Comment by OpenSteveMap** [comment]
should be ready now

File: examples/map-component/index.html [hunk 4]
```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Yaga - Leaflet-NG2 | Tile-Layer Example</title>
+    <title>Yaga - Leaflet-NG2 | Map-Component Example</title>
```
Comment (atd-schubert) [inline examples/map-component/index.html:6]: I've made an issue to correct the spelling of YAGA #144

**Review by atd-schubert — APPROVED** [review]

**Pull request merged**